mojaloop/values.yaml

```diff
@@ -6662,12 +9168,27 @@ mojaloop-ttk-simulators:
               host: 'ttksim1.local'
 
         config:
-          ## TODO: Update the below line once the rules have been migrated to https://github.com/mojaloop/sdk-scheme-adapter 
-          rules_response__default.json: 'https://github.com/mojaloop/testing-toolkit-test-cases/raw/v14.1.0/rules/sdk-bulk/response_rules.json'
-          # rules_response__default.json: 'https://github.com/mojaloop/sdk-scheme-adapter/raw/v21.4.0/test/func/config/ttk-ttksim1/spec_files/rules_response/default.json'
-          api_definitions__mojaloop_simulator_sim_1.4__api_spec.yaml: 'https://github.com/mojaloop/sdk-scheme-adapter/raw/v21.4.0/test/func/config/ttk-ttksim1/spec_files/api_definitions/mojaloop_simulator_sim_1.4/api_spec.yaml'
-          api_definitions__mojaloop_sdk_outbound_scheme_adapter_1.0__api_spec.yaml: 'https://github.com/mojaloop/sdk-scheme-adapter/raw/v21.4.0/test/func/config/ttk-ttksim1/spec_files/api_definitions/mojaloop_sdk_outbound_scheme_adapter_1.0/api_spec.yaml'
-          api_definitions__mojaloop_sdk_outbound_scheme_adapter_1.0__callback_map.json: 'https://github.com/mojaloop/sdk-scheme-adapter/raw/v21.4.0/test/func/config/ttk-ttksim1/spec_files/api_definitions/mojaloop_sdk_outbound_scheme_adapter_1.0/callback_map.json'
+          mongodb:
+            host: *TTK_MONGO_HOST
+            port: *TTK_MONGO_PORT
+            user: *TTK_MONGO_USER
+            ## Secret-Management
+            ### Set this if you are using a clear password configured in the config section
+            password: *TTK_MONGO_PASSWORD
```

Comment:
```suggestion
            password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config.